uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/instances/leave-request.instance.js

```diff
@@ -132,12 +197,16 @@ define([
          */
         calculateBalanceChange: function (calculationUnit) {
           var params = ['contact_id', 'from_date', 'to_date', 'type_id', 'from_date_type', 'to_date_type'];
+          var request = this;
```

Comment:
We dont need to create another variable for request, we can use `bind` below to keep using `this`. cc @AkA84

Reply:
done.